.chloggen/lokiexporter-dots-to-underscores-label-name-normalization.yaml

```diff
@@ -0,0 +1,17 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: translator/loki
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Added normalization so label name to follow Prometheus label names standard
+
+# One or more tracking issues related to the change
+issues: [14113]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: |
+  Loki doesn't support label names containing dots. After normalization, all dots in label names are replaced with underscores
```

Comment:
Shall we add a sentence telling users that they can drop relabeling from their configuration and that now the `.` -> `_` will happen automatically.

Reply:
thank you, fixed